test/account_test.go

```diff
@@ -0,0 +1,383 @@
+package blockfrostgo_test
+
+import (
+	"context"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+	"time"
+
+	blockfrostgo "github.com/blockfrost/blockfrost-go/pkg/api"
```

Comment:
We'd like to keep the import path concise hence currently it is imported from `github.com/blockfrost/blockfrost-go` under blockfrost package